origin-dapp/public/schemas/announcements.json

```diff
@@ -3,6 +3,14 @@
   "type": "object",
   "required": ["name","category","description","price"],
   "properties": {
+    "listingType": {
```

Comment:
@franckc Does this mean that all listings of schema "services", for example, will have to be fractional? It's looking like we're going to be able to have a one-to-one mapping between schema and `listingType` based on our consensus (pre-approval from J&M). But that is not likely to be true in the future.

Reply:
Yeah that's a good point @micahalcorn - my original thinking was to make it an `enum` with a default so a dapp developer could override it if they want, but after @franckc 's comment, I figured it would be just as easy (and maybe cleaner) for them to change a `const` than override an enum's default... maybe it's about the same really.

Reply:
Late to the party, but I agree with having a single listingType per schema. 

It seems clear we'll eventually need to go to having one jsonschema per listing type in the future, e.g. one for bikes with size/color/make, one for homesharing with bedrooms/baths. As the jsonschema files get more specific, we certainly only want one 'listingType' per schema.